bigbluebutton-html5/imports/ui/components/video-provider/component.jsx

```diff
@@ -990,20 +1192,37 @@ class VideoProvider extends Component {
       this.stopWebRTCPeer(message.cameraId);
     }
 
-    this.logger('error', `Handle error ---------------------> ${message.message}`, 'video_provider_handle_sfu_error', { message });
+    logger.error({
+      logCode: 'video_provider_handle_sfu_error',
+      extraInfo: {
+        topic: 'video',
+        message,
+      },
+    }, 'Handle error');
   }
 
   shareWebcam() {
     if (this.connectedToMediaServer()) {
-      this.logger('info', 'Sharing webcam', 'video_provider_share_webcam');
+      logger.info({
+        logCode: 'video_provider_share_webcam',
+        extraInfo: {
+          topic: 'video',
+        },
+      }, 'Sharing webcam');
       this.sharedWebcam = true;
       VideoService.joiningVideo();
     }
   }
 
   unshareWebcam() {
     const { userId } = this.props;
-    this.logger('info', 'Unsharing webcam', 'video_provider_unshare_webcam');
+    logger.info({
+      logCode: 'video_provider_unshare_webcam',
+      extraInfo: {
+        topic: 'video',
+      },
+    }, 'Unsharing webcam');
+    this.sharedWebcam = true;
```

Comment:
Was this supposed to be added?

Reply:
Nice catch! :sweat_smile: copy-paste leftover